### PR TITLE
Optimize description and add missing links

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

- Optimize description issues, add multiple missing links brought by #3276 and #3253 
- Change 热点可视化 (deprecated now) to 流量可视化

**NOTE:** Changes for `blacklist-control-plan.md` also apply to v3.1 and v3.0. I'll update them in another PR.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: blacklist-control-plan.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ category: performance
...
...man/5.7/en/json-search-functions.html#function_json-extract),<br/> [JSON_UNQUOTE(json_val)](https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-unquote),<br/> [JSON_OBJECT(key, val[, key, val] ...)](https://dev.mysql.com/doc/refman/5.7/en/json-creation-functions.html#function_json-object),<br/> [JSON_ARRAY([val[, val] ...])](https://dev.mysql.com/doc/refman/5.7/en/json-creation-functions.html#function_json-array),<br/> [JSON_MERGE(json_doc, json_doc[, json_doc] ...)](https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-merge),<br/> [JSON_SET(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-set),<br/> [JSON_INSERT(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-insert),<br/> [JSON_REPLACE(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-replace),<br/> [JSON_REMOVE(json_doc, path[, path] ...)](https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-remove) |
```
Comment (lilin90) [inline blacklist-control-plan.md:82]: Add missing links back.

File: blacklist-control-plan.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ category: performance
 |**优化规则**|**规则名称**|**简介**|
```
Comment (lilin90) [inline blacklist-control-plan.md:17]: The file name of this file is `blocklist-control-plan.md`, added in https://github.com/pingcap/docs/pull/3043. I'll rename it later to keep zh and en consistent for easy maintenance.

File: blacklist-control-plan.md [hunk 3]
```diff
@@ -225,7 +225,7 @@ desc mysql.expr_pushdown_blacklist;
     Query OK, 0 rows affected (0.00 sec)
     ```
 
-5. 重新观察执行计划，`<` 和 `>` 表达式又可以重新被下推到 TiKV Coprocessor。
+5. 重新观察执行计划，可以看到只有 `>` 表达式被重新下推到 TiKV Coprocessor，`<` 表达式仍然被禁用下推。
```
Comment (lilin90) [inline blacklist-control-plan.md:228]: @lzmhhh123 PTAL

File: blacklist-control-plan.md (hunk 2 above)
Comment (ran-huang) [reply blacklist-control-plan.md:17]: Got it. Thanks!

**Pull request merged**